src/main/java/com/uid2/admin/vertx/service/SharingService.java

```diff
@@ -75,43 +75,81 @@ private void handleSetKeyset(RoutingContext rc) {
                 keysetProvider.loadContent();
                 siteProvider.loadContent();
             } catch (Exception e) {
-                LOGGER.error("Failed to load key acls");
+                LOGGER.error("Failed to load keysets");
                 rc.fail(500);
             }
 
+            final Map<Integer, Keyset> keysetsById = this.keysetProvider.getSnapshot().getAllKeysets();
+
             final JsonObject body = rc.body().asJsonObject();
 
             final JsonArray whitelist = body.getJsonArray("allowed_sites");
```

Comment:
whitelist => allowedSites

Reply:
I think this is part of another PR

Reply:
that PR is merged already and it only changed the all the json, not all the java variable names